core/src/test/scala/integration/kafka/admin/ConfigCommandIntegrationTest.scala

```diff
@@ -26,15 +26,17 @@ import org.apache.kafka.common.config.ConfigException
 import org.apache.kafka.common.network.ListenerName
 import org.apache.kafka.common.security.auth.SecurityProtocol
 import org.junit.jupiter.api.Assertions._
-import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.ValueSource
 
 import scala.collection.Seq
 import scala.jdk.CollectionConverters._
 
 class ConfigCommandIntegrationTest extends QuorumTestHarness with Logging {
 
-  @Test
-  def shouldExitWithNonZeroStatusOnUpdatingUnallowedConfigViaZk(): Unit = {
+  @ParameterizedTest
+  @ValueSource(strings = Array("zk"))
+  def shouldExitWithNonZeroStatusOnUpdatingUnallowedConfigViaZk(quorum: String): Unit = {
```

Comment:
Can you confirm that you need a ZK cluster to perform this test? If not can we move it to the suite that doesn't inherent from `QuorumTestHarness`?

Reply:
I confirmed locally, this test case will timeout if we provide no zk cluster.